src/client/Microsoft.Identity.Client/TelemetryCore/Internal/Events/ApiEvent.cs

```diff
@@ -123,12 +123,7 @@ public string CacheInfoString
 
         public long DurationInCacheInMs { get; set; }
 
-        public TokenType? TokenType { get; set; }
-
-        public string TokenTypeString
-        {
-            get => TokenType.HasValue ? TokenType.Value.ToString("D") : null;
-        }
+        public string TokenType { get; set; }
```

Comment:
This is a breaking change. The current value that is recorded in the server side telemetry is a numeric value of the enum. For example: `5|1004,2,,,|0,1,1`. This change will break all the queries checking this value.

Reply:
Discussed offline, All tokenTypes will be represented as an int